javascript-source/handlers/subscribeHandler.js

```diff
@@ -383,12 +485,27 @@
             $.say($.whisperPrefix(sender) + $.lang.get('subscribehandler.giftsub.msg.set'));
         }
 
+        /*
+         * @commandpath giftanonsubmessage[message] - Set a message
+         for resubscribers.
+         */
+        if (command.equalsIgnoreCase('giftanonsubmessage')) {
+            if (action === undefined) {
+                $.say($.whisperPrefix(sender) + $.lang.get('subscribehandler.giftanonsub.msg.usage'));
+                return;
+            }
+
+            giftAnonSubMessage = argsString;
+            $.setIniDbString('subscribeHandler', 'giftAnonSubMessage', giftAnonSubMessage);
+            $.say($.whisperPrefix(sender) + $.lang.get('subscribehandler.giftanonsub.msg.set'));
+        }
+
         /*
          * @commandpath massgiftsubmessage [message] - Set a message for resubscribers.
```

Comment:
This is on me, but if you can update it as well. 

`Set a message for gifting alerts.`